docs/design/extending-api.md

```diff
@@ -113,18 +113,17 @@ For example, if a user creates:
 
 ```yaml
 metadata:
-  name: cron-tab.example.com
+  name: cron-tab.stable.example.com
 apiVersion: experimental/v1alpha1
 kind: ThirdPartyResource
 description: "A specification of a Pod to run on a cron style schedule"
 versions:
-  - name: stable/v1
-  - name: experimental/v2
+- v1
```

Comment:
Another thing I noticed: there is an APIGroup field in versions, https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/experimental/types.go#L177

The comment says it specifies "the API group to add this object into, default 'experimental'."  Does that make sense? Isn't the group already specified in "name: cron-tab.stable.example.com"?